### PR TITLE
Handle deletes: delete answer

### LINKED ISSUE #55: deleting !grol input message doesn’t delete the reply

### PR DESCRIPTION
**Pull request by ldemailly**
- #55

fixes #55

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -338,6 +339,20 @@ func updateMessage(session *discordgo.Session, message *discordgo.MessageUpdate)
...
+	reply, found := msgSet.Get(message.ID)
+	if !found {
+		log.S(log.Debug, "message not handled before", log.Any("id", message.ID))
+		return
+	}
+	log.S(log.Info, "message delete detected",
+		log.Any("id", message.ID),
+		log.Any("reply", reply),
+		log.Any("before", message.BeforeDelete))
+	handleMessage(session, &discordgo.MessageCreate{Message: message.Message}, reply)
```
Comment (ccoVeille) [inline bot.go:353]: So a message deletion? Call a messageCreate ?

I'm asking because it looks weird

Comment (ldemailly) [reply bot.go:353]: create, update, and now delete share the same logic, which is that create doesn’t have a reply already and update/delete both (can for update, does for delete) delete replies

it’s a bit overkill but also avoids duplication

Comment (ldemailly) [reply bot.go:353]: I guess I could probably pass only the inner message to handle message

Comment (ldemailly) [reply bot.go:353]: #63

**Pull request merged**